test/cfg/std.c

```diff
@@ -162,7 +162,6 @@ void nullpointer(int value)
 
     // #6100 False positive nullPointer - calling mbstowcs(NULL,)
     res += mbstowcs(0,"",0);
-    // TODO cppcheck-suppress unreadVariable
```

Comment:
why is this TODO removed?

Reply:
The return value is saved in the variable res. This variable is used later, so there is correctly no longer an unreadVariable warning issued by Cppcheck. It seems to be a relict from the time where the library checks did not work properly and when they worked again the suppression was no longer correct.
With commit 58034dee869677cb0c612b61f4e0e7c620bc6b08 you marked it with TODO because by fixing the library tests this suppression caused an error.

Reply:
ah yes.. on line 177.. I missed that one. Good catch.